<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>711</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>491</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>驾驶员理论课程模拟考试与学习系统</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputUserID">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUserLogin">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户登录</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>管理员密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputAdminPwd">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdminLogin">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>190</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>管理员登陆</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QListWidget" name="listQuestions">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>471</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="editQuestion">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="editA">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="editB">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="editC">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>170</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="editD">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="editAnswer">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdd">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>56</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>添加试题</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDelete">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>56</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>删除试题</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpdate">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>56</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>保存修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStats">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>56</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>查看统计</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogoutAdmin">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>290</y>
        <width>56</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QTableWidget" name="tableScores">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>351</width>
        <height>321</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>考试模式</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>分数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>考试时间</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>按用户筛选：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboFilter">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>60</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRefresh">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBackToAdmin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>返回试题管理页</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>按模式筛选：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboModeFilter">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>60</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>621</width>
        <height>281</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>在线考试</string>
       </attribute>
       <widget class="QLabel" name="labelExamQuestion">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>531</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>题目</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioA">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>411</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>A</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioB">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>401</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioC">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>411</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioD">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>451</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>D</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnNext">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>190</y>
          <width>56</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>下一题</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSubmit">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>220</y>
          <width>56</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>交卷</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>知识查询</string>
       </attribute>
       <widget class="QLineEdit" name="inputSearch">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>321</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>输入关键词</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSearch">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>20</y>
          <width>56</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textResult">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>321</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="frameFreeSetup">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>训练题数：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="inputFreeNum">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGenFree">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>10</y>
         <width>56</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>生成题目</string>
       </property>
      </widget>
     </widget>
     <widget class="QLCDNumber" name="lcdTimer">
      <property name="geometry">
       <rect>
        <x>513</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::SegmentStyle::Flat</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReturnToMode">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>260</y>
        <width>56</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogout">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>290</y>
        <width>56</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="btnModeMock">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>模拟考试</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnModeFree">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>160</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>自由训练</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="btnNextWrong">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>150</y>
        <width>56</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>下一题</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioWrongA">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioWrongD">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioWrongC">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioWrongB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblWrongQuestion">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>531</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>题目</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExitWrong">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>200</y>
        <width>56</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblCorrectAnswer">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green; font-weight: bold; font-size: 16px;</string>
      </property>
      <property name="text">
       <string>正确答案</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
